templates/strings/objc-m.stencil

```diff
@@ -0,0 +1,80 @@
+// Generated using SwiftGen — https://github.com/SwiftGen/SwiftGen
+
+{% if tables.count > 0 %}
+
+#import "Localizable.h"
+
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wformat-security"
+
+@interface BundleToken : NSObject
+@end
+
+@implementation BundleToken
+@end
+
+NSString* tr(NSString* tableName, NSString* key, ...) {
+    va_list args;
+    va_start(args, key);
+
+    NSBundle *bundle = [NSBundle bundleForClass:BundleToken.class];
```

Comment:
Nitpick: add a space between the `:` and the argument (here and the lines below).

Reply:
really?  i haven't seen that style elsewhere.

Reply:
From taking a look at most style guides, seems I have a weird style 😆 Keep it as is for this.

Reply:
avec pleasure!